app/views/guidance/recommended-pig-labs.njk

```diff
@@ -0,0 +1,134 @@
+{% extends './layouts/layout.njk' %}
+{% block pageTitle %}Guidance for vets - {{ serviceName }}{% endblock %}
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-quarter">
+      <div class="govuk-!-margin-bottom-6 gem-c-organisation-logo brand--department-for-environment-food-rural-affairs">
+      <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs">
+        <span class="gem-c-organisation-logo__name">Department<br>for Environment<br>Food &amp; Rural Affairs</span>
+      </a>
+    </div>
+    </div>
+    <div class="govuk-grid-column-one-quarter">
+      <div class="govuk-!-margin-bottom-6 gem-c-organisation-logo brand--rural-payments-agency">
+      <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="https://www.gov.uk/government/organisations/rural-payments-agency">
+        <span class="gem-c-organisation-logo__name">Rural Payments <br>Agency</span>
+      </a>
+      </div>
+    </div>
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <div class="govuk-panel govuk-panel--confirmation govuk-panel--blue govuk-!-text-align-left govuk-!-margin-bottom-7">
+        <div id= "guidance" class="govuk-panel__body govuk-!-margin-bottom-3">
+          Guidance
+        </div>
+        <h1 class="govuk-heading-xl">Recommended laboratories to test for porcine reproductive and respiratory syndrome (PRRS) in pigs</h1>
+        <div class="govuk-panel__body">
+          Published 22 June 2023
+        </div>  
+      </div>  
+
+      <section class="gem-c-devolved-nations">
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
+          Applies to England
```

Comment:
There is a lot of whitespace above and below the `Applies to England` text which isn't present in the prototype. Happy for it to be left as is, but just think it would look better without it. This also applies to the cattle and sheep pages.